### PR TITLE
update examples project

### PR DESCRIPTION
**Pull request by rozaev**
Hello @dylans and @jason0x43!

I tried to update or demo projects and I almost succeeded.

Right now I have problems with 2 examples:
ember-example and dojo-example (so ironic).

All other examples works fine.
In my humble opinion we should delete ember-example and dojo-example for now.
Non-existing dojo example better than broken dojo example.

Next step:
If I right understand, all examples was taken from there:
https://github.com/tastejs/todomvc
I take actual examples from there. Then we got ember-example and dojo-example.

What do you think?
About code changes: I do all alright or there some terrible mistake in my changes?

Just say what I should do and I will do it.

### CODE REVIEW

**Comment by rozaev** [comment]
Okay, I fix the dojo example, but ember still has problems with depends.

**Comment by rozaev** [comment]
Maybe, I should add code coverage report to all examples? Not only to backbone?

**Comment by rozaev** [comment]
This pull-request fix issues:
https://github.com/theintern/intern-examples/issues/36
https://github.com/theintern/intern-examples/issues/34

File: parallel-example/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+jquery-example
```
Comment (dylans) [inline parallel-example/README.md]: parallel-example ?

File: parallel-example/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "angularjs-intern-example",
```
Comment (dylans) [inline parallel-example/package.json]: parallel-intern-example ?

File: parallel-example/package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  "version": "0.1.0",
+  "description": "An example showing a typical AngularJS application tested with Intern",
```
Comment (dylans) [inline parallel-example/package.json]: update description?

File: react-example/README.md [hunk 4]
```diff
@@ -1,38 +1,34 @@
-# React TodoMVC Example
+jquery-example
```
Comment (dylans) [inline react-example/README.md]: react-example

File: react-example/tests/intern.js [hunk 5]
```diff
@@ -52,13 +38,78 @@ define({
...
 
-	// A regular expression matching URLs to files that should not be included in code coverage analysis
-	excludeInstrumentation: /^tests|bower_components|node_modules\//
-});
\ No newline at end of file
+	// A regular expression matching URLs to files that should not be included in code coverage analysis. Set to `true`
+	// to completely disable code coverage.
+	excludeInstrumentation: /^(?:tests|bower_components|node_modules)\//
+});
+//
```
Comment (dylans) [inline react-example/tests/intern.js]: Remove this commented out section

File: angularjs-example/package.json [hunk 6]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
```
Comment (dylans) [inline angularjs-example/package.json]: Maybe bump the version numbers to 0.2.0? :)

File: angularjs-example/README.md [hunk 7]
```diff
@@ -1,20 +1,34 @@
 angularjs-example
```
Comment (rozaev) [inline angularjs-example/README.md]: done

File: angularjs-example/README.md [hunk 8]
```diff
@@ -1,20 +1,34 @@
...
-
+=============
 ## Setup
 
-```
-npm install -g bower
-npm install
-bower install
-```
+1. This demo runs with local Selenium.
```
Comment (rozaev) [inline angularjs-example/README.md]: done

File: angularjs-example/tests/intern.js [hunk 9]
```diff
@@ -1,60 +1,48 @@
-// Learn more about configuring this file at <https://github.com/theintern/intern/wiki/Configuring-Intern>.
-// These default settings work OK for most people. The options that *must* be changed below are the
-// packages, suites, excludeInstrumentation, and (if you want functional tests) functionalSuites.
+// Learn more about configuring this file at <https://theintern.github.io/intern/#configuration>.
+// These default settings work OK for most people. The options that *must* be changed below are the packages, suites,
+// excludeInstrumentation, and (if you want functional tests) functionalSuites
```
Comment (rozaev) [inline angularjs-example/tests/intern.js]: I guess we shouldn't mention benchmarkSuites there. It is a pretty new functional, I guess we should write a new example for benchmarks.

File: parallel-example/README.md (hunk 1 above)
Comment (rozaev) [inline parallel-example/README.md]: done. copy-paste is evil.

File: parallel-example/package.json (hunk 2 above)
Comment (rozaev) [inline parallel-example/package.json]: done. copy-paste is evil.

File: parallel-example/package.json (hunk 3 above)
Comment (rozaev) [inline parallel-example/package.json]: done - "An example showing a typical Angular 1.x application tested with Intern"

File: react-example/README.md (hunk 4 above)
Comment (rozaev) [inline react-example/README.md]: done. copy-paste is evil.

File: react-example/tests/intern.js (hunk 5 above)
Comment (rozaev) [inline react-example/tests/intern.js]: done.

File: angularjs-example/package.json (hunk 6 above)
Comment (rozaev) [inline angularjs-example/package.json]: done. It is a great honor)

**Comment by rozaev** [comment]
@dylans Feedback is had processed.

File: angularjs-example/tests/intern.js (hunk 9 above)
Comment (dylans) [inline angularjs-example/tests/intern.js]: Agreed, we can do that elsewhere.

File: angularjs-example/README.md (hunk 8 above)
Comment (jason0x43) [inline angularjs-example/README.md]: Just to clarify, SeleniumTunnel doesn't use selenium-standalone.

Comment (rozaev) [inline angularjs-example/README.md]: @jason0x43 please explain, i didn't read the source code, I just read documentation there: https://theintern.github.io/intern/ 
> 3.3SeleniumTunnel – For loading and managing a local standalone Selenium server

Comment (rozaev) [inline angularjs-example/README.md]: Oh, I understand. You mean this selenium-standalone: https://www.npmjs.com/package/selenium-standalone

Yes, I understand that.

**Comment by jason0x43** [comment]
This has been merged. Thanks!

**Pull request closed without merging**